plugins/de.ovgu.featureide.fm.ui/src/de/ovgu/featureide/fm/ui/editors/featuremodel/operations/CreateFeatureBelowOperation.java

```diff
@@ -59,9 +67,48 @@ protected FeatureIDEEvent operation() {
 		feature = featureModel.getFeature(feature.getName());
 		feature.getStructure().addChild(newFeature.getStructure());
 
+		newGraphicalFeature = graphicalFeature.getGraphicalModel().getGraphicalFeature(newFeature);
+
+		if (graphicalFeature.getGraphicalModel().getLayout().getLayoutAlgorithm() == 0) {
+			setNewPositionFeatureBelow();
+		}
+
 		return new FeatureIDEEvent(featureModel, EventType.FEATURE_ADD, feature, newFeature);
 	}
 
+	/**
+	 * adjust position of the new feature in manual layout
+	 */
+	private void setNewPositionFeatureBelow() {
```

Comment:
Declare const with descriptive names instead of random integer values (Magic Number Anti-Pattern)